src/libs/KeyboardAvoidingView/index.js

```diff
@@ -14,18 +11,9 @@ const defaultProps = {
     children: null,
 };
 
-function KeyboardAvoidingView({children}) {
-    return (
-        <KeyboardAvoidingViewComponent
-            enabled={false}
-            behavior="height"
-            style={[styles.w100, styles.h100]}
-        >
-            {children}
-        </KeyboardAvoidingViewComponent>
-    );
-}
+const KeyboardAvoidingView = props => props.children;
 
 KeyboardAvoidingView.propTypes = propTypes;
 KeyboardAvoidingView.defaultProps = defaultProps;
+KeyboardAvoidingView.displayName = 'KeyboardAvoidingView';
```

Comment:
It looks like the index.ios.js counterpart doesn't have the displayName set, can we set it there as well?

Reply:
adding